common/dma/gs.cpp

```diff
@@ -352,15 +353,17 @@ std::string GsTexa::print() const {
 std::string GsTex0::print() const {
   return fmt::format(
       "tbp0: {} tbw: {} psm: {} tw: {} th: {} tcc: {} tfx: {} cbp: {} cpsm: {} csm: {}\n", tbp0(),
-      tbw(), psm(), tw(), th(), tcc(), tfx(), cbp(), cpsm(), csm());
+      tbw(), fmt::underlying(psm()), tw(), th(), tcc(), fmt::underlying(tfx()), cbp(), cpsm(),
+      csm());
 }
 
 std::string GsPrim::print() const {
-  return fmt::format("0x{:x}, kind {}\n", data, kind());
+  return fmt::format("0x{:x}, kind {}\n", data, fmt::underlying(kind()));
 }
 
 std::string GsFrame::print() const {
-  return fmt::format("fbp: {} fbw: {} psm: {} fbmsk: {:x}\n", fbp(), fbw(), psm(), fbmsk());
+  return fmt::format("fbp: {} fbw: {} psm: {} fbmsk: {:x}\n", fbp(), fbw(), fmt::underlying(psm()),
```

Comment:
I'm a little confused by how this code ever worked... I thought `libfmt` would complain if you tried to print an enum directly.